clojure-mode.el

```diff
@@ -3129,7 +3129,7 @@ Assumes cursor is at beginning of function."
   "Add an arity to a function.
 
 Assumes cursor is at beginning of function."
-  (let ((beg-line (what-line))
+  (let ((beg-line (line-number-at-pos))
```

Comment:
Without this change 3 tests for `clojure-add-arity` were failing with an additional \n in the generated output. Only happens when running tests with eldev.(Might be because `what-line` is an interactive fn with side effects)

Reply:
Sounding good. Please confirm that you've run this modified version locally

Reply:
I have run it locally and it works fine. 
Locally both the versions work but not using 'what-line' is still better.